docs/pages/linking/share-content-into-your-app.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Receiving content to your app
+sidebar_title: Receiving content to your app
+description: Learn how to receive content to your app from another app.
+---
+
+import { FileTree } from '~/ui/components/FileTree';
+
+You may want to let other apps share content into your app &mdash; like articles, images, or files &mdash; using the native system share sheet. This guide explains how that works under the hood and gives an overview of the tools available to help you implement it.
+
+With [`expo-sharing`](/versions/latest/sdk/sharing), you can already share content _from_ your app into other apps. Receiving content works the other way around: your app needs to become a share target, which on Android means responding to Intents and on iOS means handling a Share Extension.
+
+## OS foundations
+
+**Android**
+
+On Android, the system delivers shared data to an Activity. An Activity is a single, focused screen in your app, and it can also act as an entry point that the system launches directly when the user selects your app in the share sheet.
+
+- Apps register intent filters (ACTION_SEND / ACTION_SEND_MULTIPLE) for specific MIME types (e.g., text/plain, image/\*) in AndroidManifest.xml.
```

Comment:
```suggestion
- Apps register intent filters (`ACTION_SEND`/`ACTION_SEND_MULTIPLE`) for specific MIME types (for example, text/plain, image/\*) in **AndroidManifest.xml**.
```

- Let's highlight the file name: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#file-names-directory-names-file-extensions-as-bold-text
- Let's use inline syntax to highlight native intents.